hack/update_catalog.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
This won't be required in release branches. Having the fbc related content in main branch alone should be enough.